nano_graphrag/_llm.py

```diff
@@ -4,6 +4,7 @@
 
 import aioboto3
 from openai import AsyncOpenAI, AsyncAzureOpenAI, APIConnectionError, RateLimitError
+from azure.core.exceptions import HttpResponseError
```

Comment:
You should add `azure` dep in `requirement.txt`

Reply:
wait there is no Azure in requirement? shiii,sorry for that